npm install error

as requested https://github.com/miraclx/freyr-js/issues/39#issuecomment-781437631:

$ npm install -g miraclx/freyr-js
npm ERR! code 1
npm ERR! git dep preparation failed
npm ERR! command /usr/local/Cellar/node/15.8.0/bin/node /usr/local/lib/node_modules/npm/bin/npm-cli.js install --force --cache=/Users/bfm/.npm --prefer-offline=false --prefer-online=false --offline=false --no-progress --no-save --no-audit
npm ERR! npm WARN using --force Recommended protections disabled.
npm ERR! npm ERR! code 1
npm ERR! npm ERR! path /Users/bfm/.npm/_cacache/tmp/git-clone-5f432a76
npm ERR! npm ERR! command failed
npm ERR! npm ERR! command sh -c node ./scripts/download.js
npm ERR! npm ERR! node:internal/modules/cjs/loader:928
npm ERR! npm ERR! throw err;
npm ERR! npm ERR! ^
npm ERR! npm ERR!
npm ERR! npm ERR! Error: Cannot find module 'got'
npm ERR! npm ERR! Require stack:
npm ERR! npm ERR! - /Users/bfm/.npm/_cacache/tmp/git-clone-5f432a76/lib/downloader.js
npm ERR! npm ERR! - /Users/bfm/.npm/_cacache/tmp/git-clone-5f432a76/scripts/download.js
npm ERR! npm ERR! at Function.Module._resolveFilename (node:internal/modules/cjs/loader:925:15)
npm ERR! npm ERR! at Function.Module._load (node:internal/modules/cjs/loader:769:27)
npm ERR! npm ERR! at Module.require (node:internal/modules/cjs/loader:997:19)
npm ERR! npm ERR! at require (node:internal/modules/cjs/helpers:92:18)
npm ERR! npm ERR! at Object. (/Users/bfm/.npm/_cacache/tmp/git-clone-5f432a76/lib/downloader.js:5:13)
npm ERR! npm ERR! at Module._compile (node:internal/modules/cjs/loader:1108:14)
npm ERR! npm ERR! at Object.Module._extensions..js (node:internal/modules/cjs/loader:1137:10)
npm ERR! npm ERR! at Module.load (node:internal/modules/cjs/loader:973:32)
npm ERR! npm ERR! at Function.Module._load (node:internal/modules/cjs/loader:813:14)
npm ERR! npm ERR! at Module.require (node:internal/modules/cjs/loader:997:19) {
npm ERR! npm ERR! code: 'MODULE_NOT_FOUND',
npm ERR! npm ERR! requireStack: [
npm ERR! npm ERR! '/Users/bfm/.npm/_cacache/tmp/git-clone-5f432a76/lib/downloader.js',
npm ERR! npm ERR! '/Users/bfm/.npm/_cacache/tmp/git-clone-5f432a76/scripts/download.js'
npm ERR! npm ERR! ]
npm ERR! npm ERR! }

--- a/package.json
+++ b/package.json
@@ -84,7 +84,7 @@
     "tough-cookie": "^4.0.0",
     "xbytes": "^1.6.2",
     "xprogress": "^0.17.3",
-    "youtube-dl": "github:miraclx/node-youtube-dl",
+    "youtube-dl": "^3.0.2",
     "yt-search": "^2.5.1"
   },
   "devDependencies": {}

--- a/package.json
+++ b/package.json
@@ -84,7 +84,7 @@
     "tough-cookie": "^4.0.0",
     "xbytes": "^1.6.2",
     "xprogress": "^0.17.3",
-    "youtube-dl": "^3.0.2",
+    "youtube-dl": "^3.4.0",
     "yt-search": "^2.5.1"
   },
   "devDependencies": {}

